JSQMessagesViewController/Factories/JSQMessagesAvatarImageFactory.h

```diff
@@ -36,7 +38,7 @@
 *
 *  @return An initialized `JSQMessagesAvatarImage` object if created successfully, `nil` otherwise.
 */
-+ (JSQMessagesAvatarImage *)avatarImageWithPlaceholder:(UIImage *)placeholderImage diameter:(NSUInteger)diameter;
++ (nullable JSQMessagesAvatarImage *)avatarImageWithPlaceholder:(UIImage *)placeholderImage diameter:(NSUInteger)diameter;
```

Comment:
same as above, should actually be nonnull. let's remove nullable here and update docs.